contracts/registry/IdentityRegistry.sol

```diff
@@ -173,6 +175,15 @@ contract IdentityRegistry is IIdentityRegistry, AgentRole {
             }
             for (uint256 j = 0; j < claimIds.length; j++) {
                 (foundClaimTopic, scheme, issuer, sig, data, ) = identity(_userAddress).getClaim(claimIds[j]);
+
+                if (
+                    IClaimIssuer(issuer).isClaimValid(identity(_userAddress), requiredClaimTopics[claimTopic], sig, data)
+                    && tokenIssuersRegistry.hasClaimTopic(issuer, requiredClaimTopics[claimTopic])
+                    && tokenIssuersRegistry.isTrustedIssuer(issuer)
+                ) {
+                    j = claimIds.length;
+                }
+
```

Comment:
Mh, this is a bit weird, a few things to test:
- We should probably encapsulate this in a try/catch, or a invalid claim added on an Identity that revert because of a wrong ClaimIssuer implementation could block an Identity from being validated via another claim. (Minor severity, as the identity owner could remove the problematic claim first).
- The `isClaimValid` method on the ClaimIssuer is called twice in the loop (cf below at line 194). We could probably remove one of these two calls or at least store the result in a temporary memory variable.

Reply:
i added a try/catch and a separated variable for the validity to avoid calling it twice